### ISSUE TITLE
Backend: Add Facebook Pages subscriptions

### ISSUE THREAD

**Issue by samuelclay** [samuelclay/NewsBlur#1089]
Similar to Youtube and Twitter API integration, use the Facebook OAuth credentials to allow people to subscribe to Facebook Pages.

Ideally we'd add an "Advanced" groupset in the Add Site dialog where somebody could enter in a YouTube url, a Twitter url, or a Facebook Group/Pages url.

**Comment by starsep** [comment]
I'm interested in this feature and I'd to work on it if that's possible.
I would use this API https://developers.facebook.com/docs/graph-api/reference/v8.0/page/feed

What is difference between this issue and #812? Is it lack of user credentials? 
In theory one could parse HTML, example: https://mbasic.facebook.com/newsblur?v=timeline&filter=2&refid=17 and make RSS feed out it. Then there is no need for OAuth credentials in the case of public Facebook pages.
Due to Facebook anti-scraping restrictions and limiting traffic from the same IP, using a proper API seems like a better solution.

**Comment by samuelclay** [comment]
You're right, this ticket and #812 are the same. I'll close #812 in lieu of this ticket.

I actually built it but it doesn't work. You can see the facebook_fetcher.py file, but I can't get it to work for some reason and haven't returned to it. Probably something to do with API credentials.
